ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh

```diff
@@ -80,25 +80,25 @@ CUSTOM_KUBEOBJECTS_PATH="${CUSTOM_KUBEOBJECTS_PATH}${KEYWORD_CUSTOM_KUBEOBJECTS}
 # check if the directory exists or not in first place 
 # if not send message and gracefully exit.
 
-LIST_OF_KUBEOBJECTS="${LIST_OF_KUBEOBJECTS:=''}"
-
 if [ ! -d "${CUSTOM_KUBEOBJECTS_PATH}" ]
 then
     echo "Directory ${CUSTOM_KUBEOBJECTS_PATH} DOES NOT exists."
     echo "Please check the vault"
     echo "The following step gracefully exits now"
     exit 0 
+fi
+
+echo "The ${CUSTOM_KUBEOBJECTS_PATH} found"
```

Comment:
Suggest: `echo "CUSTOM_KUBEOBJECTS_PATH found: $CUSTOM_KUBEOBJECTS_PATH"`

Reply:
Will change it.